testing/py-biopython/APKBUILD

```diff
@@ -0,0 +1,42 @@
+# Contributor: Charles Pritchard <chuck@jumis.com>
+# Maintainer: Charles Pritchard <chuck@jumis.com>
+pkgname=py-biopython
+_pkgname=biopython
+pkgver=1.74
+pkgrel=0
+pkgdesc="Python tools for computational molecular biology."
+url="http://biopython.org/"
+arch="all"
+license="BSD-3-Clause"
+depends="py-numpy"
+makedepends="linux-headers py-numpy-dev python3-dev py-setuptools"
```

Comment:
py3-setuptools

Reply:
you still want this package named py-biopython?